### ISSUE TITLE
If no alt text supplied for banner, use caption

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#660]
If no alt text supplied for banner, use caption

**Comment by jonfroehlich** [comment]
added
